testing/btest/scripts/policy/frameworks/logging-debug.zeek

```diff
@@ -0,0 +1,21 @@
+# @TEST-EXEC: zeek -r ${TRACES}/socks.trace %INPUT
```

Comment:
Mind moving this file into `frameworks/analyzer` instead of `frameworks/`? (totally randomly found while skimming)